Newsletters: add 374 (2025-10-03)

bitschmidty opened the pull request:
- [x] news @harding 
- [x] changing consensus @harding 
  - [x] GSR BIPs @reardencode
- [x] releases @Gustavojfe 
- [x] notable code @Gustavojfe 
- [x] lede @bitschmidty

bitschmidty reviewed (commented):
Updated some formatting of the GSR item.

LarryRuane reviewed (commented):
A few small suggestions

reardencode commented on _posts/en/newsletters/2025-10-03-newsletter.md:
```diff
@@ -0,0 +1,111 @@
+{% include snippets/recap-ad.md when="2025-10-07 16:30" %}
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="" %}
+[rr0]: https://gnusha.org/pi/bitcoindev/877bxknwk6.fsf@rustcorp.com.au/
```
fwiw bitcoindevs.xyz seemed to be outdated when I was writing about GSR which is why I used gnusha.

harding replied:
@bitschmidty my only preference is to use a publicinbox instance rather than the google list when possible.  I don't think there's any need to commit to a particular instance.

bitschmidty commented on _posts/en/newsletters/2025-10-03-newsletter.md:
```diff
@@ -0,0 +1,194 @@
+- [LND v0.20.0-beta.rc1][] is a release candidate for a new version of this
+  popular LN node implementation that introduces multiple bug fixes, persistence
+  of node announcement settings across restarts, a new `noopAdd` [HTLC][topic
+  htlc] type, support for [P2TR][taproot] fallback addresses on [BOLT11][]
```
```suggestion
  htlc] type, support for [P2TR][topic taproot] fallback addresses on [BOLT11][]
```

bitschmidty commented on _posts/en/newsletters/2025-10-03-newsletter.md:
```diff
@@ -0,0 +1,194 @@
+  integration of Bitcoin Core with an external [Stratum v2][topic pooled mining]
+  mining service that creates, manages and submits block templates.
+
+- [Bitcoin Core ##33446][] fixes a bug introduced when the `target` field was
```
```suggestion
- [Bitcoin Core #33446][] fixes a bug introduced when the `target` field was
```

bitschmidty commented:
Pushed one more small fixup for the notable code segment. Added the lede. Squashed commits.

Thank you @reardencode for authoring the changing consensus segment this week and for @Gustavojfe for authoring the releases and notable code segments and @LarryRuane for reviewing! 🚀
